### DISCUSSION TITLE
Add Info Language selection in mass update

### DISCUSSION THREAD

**Discussion by Orion83** [pymedusa/Medusa#10513] [Ideas & Feature requests]
Hello,
Is it possible to add the possibility de change the info language with episode renaming in the mass update page?

**Comment by p0psicles** [comment]
Yeah I could take a look at that

**Reply by Orion83** [reply]
would be cool since when I reinstalled medusa on my nas, it found all my series but took automatically english info language and renamed all my episodes in english.
Hope my english is not too poor :P

**Reply by p0psicles** [reply]
It's added

**Reply by Orion83** [reply]
thanks a lot !
